### PR TITLE
Glossary page updates

### PR DESCRIPTION
**Pull request by ianmiell**

### CODE REVIEW

**Review by CharlKlein — COMMENTED** [review]
LGTM

File: docs/glossary.md [hunk 1]
```diff
@@ -5,35 +5,84 @@ sidebar_label: Glossary
...
+| Compliance Framework Owner    | The application owner of Compliance Framework.                                                                                                                                                                                                  |
+| Control                       | Policies and procedures designed to ensure systems are secure and/or stable and/or resilient. (aka Requirement, or Guideline)                                                                                                                   |
+| Control Baseline              | 'A specific set of selected security control requirements from one or more control catalogs for use in managing risks in an information system' (OSCAL Terminology). Also known as an 'Overlay'.
+                                  'the set of controls that are applicable to information or an information system to meet legal, regulatory, or policy requirements, as well as address protection needs for the purpose of managing risk' (NIST 800-37)                         |
```
Comment (robel-yemane) [inline docs/glossary.md:28]: Needs proper formatting.
Is this the definition of 'Overlay' or should it be incorporated with 'Control Baseline'?

File: docs/glossary.md [hunk 2]
```diff
@@ -5,35 +5,84 @@ sidebar_label: Glossary
...
+| Risk                          | Identifies individual risks, including weakness description, risk statement, and other risk characteristics.                                                                                                                                    |
+| Risk Level                    | Severity of risk, eg Acceptable, Critical, Low/Moderate/High Impact                                                                                                                                                                             |
+| RPO                           | Recovery Point Objective                                                                                                                                                                                                                        |
+| Senior Management Function 24 | the person in an organisation held responsible for operational failings.                                                                                                                                                                        |
```
Comment (onselakin) [inline docs/glossary.md]: A minor typo: "the"

File: docs/glossary.md [hunk 3]
```diff
@@ -5,35 +5,84 @@ sidebar_label: Glossary
...
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Assessment Runtime            | See [Assessment Runtime](/docs/architecture/assessment-runtime.md)                                                                                                                                                                              |
+| Core Services                 | See [Argus Core](/docs/architecture/core/index.md)                                                                                                                                                                                              |
+| Dashboard UI                  | See [...]                                                                                                                                                                                                                                       |
```
Comment (onselakin) [inline docs/glossary.md]: We can change this to "Portal" as the word has a broader scope aligning with the app's purpose.

File: docs/glossary.md (hunk 1 above)
Comment (onselakin) [reply docs/glossary.md:28]: Yeah, they're the same @robel-yemane: https://pages.nist.gov/OSCAL/concepts/layer/#:~:text=Organizations%20and%20system%20owners%20identify%20which%20controls%20are%20applicable%20to%20a%20system%2C%20which%20may%20include%20controls%20from%20more%20than%20one%20catalog.%20These%20are%20often%20referred%20to%20as%20a%20baseline%20or%20overlay.

**Pull request merged**